docs/guides/04-data-driven-visualizations-part-2.md

```diff
@@ -0,0 +1,113 @@
+## Data-driven visualizations (part 2)
+
+### Ramp values
+
+In the previous section we talked about using different types of input for ramp, but we always output colors picked from a list. `ramp` supports to use other types of outputs and also CARTO VL includes some fixed constant palettes of colors. Let's see it!
+
+#### Color values
+
+One way to output colors is to specify a list of colors, just like we have done in all the previous examples. This can be done with expressions like `ramp($dn, [blue, red])`. But usage of named colors (`blue`, `red`, `green`...) is not enforced, any valid color expression is ok, for example:
+- `ramp($dn, [rgb(200,220,222), rgba(200,120,22, 0.8)])`
+- `ramp($dn, [hsv(0,1,1), hsv(0.5,1,1)])`,`ramp($dn, [#00F, #F00])`
+- `ramp($dn, [blue, #F00])`
+- `ramp($dn, [opacity(blue, 0.4), opacity( #F00, 0.6),])`
+
+There is also another way to specify colors, and that is to use one of the built-in color palettes. We have built-in all the CARTOColors and ColorBrewer palettes. You can use them like this:
+- `ramp($dn, temps)`
+- `ramp($dn, tealrose)`
+- `ramp($dn, cb_blues)`
+
+The complete list of CARTOColors can be seen [here](https://carto.com/carto-colors/).
+
+Let's see all these options in actions!
+
+<div class="example-map">
+    <iframe
+        id="population-density-colors"
+        src="/developers/carto-vl/examples/maps/guides/ramp/population-density-colors.html"
+        width="100%"
+        height="1000"
+        style="margin: 20px auto !important"
+        frameBorder="0">
+    </iframe>
+    <a href="/developers/carto-vl/examples#example-population-density---colors">View my source code!</a>
+</div>
+
+#### Numeric values / Bubble-maps
+
+When dealing with point data, an interesting visualization is the bubble-map. In a bubble-map, each point has a width that depends on a feature property.
+
+Matching between numbers (the feature's data) and other numbers (the point sizes) is a special case because basic math can create the required match without the need for the special function `ramp`. However, using `ramp` facilitates some advanced usages. In the following subsections we'll see both approaches, learning how to create bubble maps like this:
+<div class="example-map">
+    <iframe
+        id="accidents-bubblemap"
+        src="/developers/carto-vl/examples/maps/guides/ramp/accidents-bubblemap.html"
```

Comment:
Something looks messy here with the styles... They look broken

Reply:
Totally agree, but I don't know how to fix it. I think this may need changes in the dev center. cc: @rochoa

Reply:
Let's create adequate issues in the https://github.com/cartodb/developers repo for anything regarding style.

Reply:
I can create an issue with this there... But it probably needs at least some changes on the CVL repo too. In any case, adding examples is a huge pain. You need this `iframe` code with a custom URL (not the same as local, there is a `maps/` path imposed by the dev center) for displaying it , another custom URL for linking in the footer (this time we need to use a `#` with a difficult to get right name). Then we have styling issues: links to production CSS on staging, the iframe style issue... And we also have our `examples.json` in CVL, which needs to be updated too, with a title and an URL (this time a local URL).

Reply:
We should definitively try to remove the /maps/ thing and fix all our links in one PR